docs/addons/nats/helm/index.md

```diff
@@ -0,0 +1,625 @@
+     Maximum Messages: unlimited
+        Maximum Bytes: unlimited
+          Maximum Age: 1y0d0h0m0s
+ Maximum Message Size: unlimited
+    Maximum Consumers: unlimited
+
+
+Cluster Information:
+
+                 Name: nats
+               Leader: nats-0
+
+State:
+
+             Messages: 2
+                Bytes: 98 B
+             FirstSeq: 1 @ 2021-09-03T08:55:39 UTC
+              LastSeq: 2 @ 2021-09-03T08:56:11 UTC
+     Active Consumers: 0
+
+sample-nats-box-785f8458d7-wtnfx:~# exit
+```
+
+Hence, we can see from the above output that the deleted data has been restored successfully from the backup.
+
+#### Resume Backup
+
+Since our data has been restored successfully we can now resume our usual backup process. Resume the `BackupConfiguration` using following command,
+
+```bash
+❯ kubectl patch backupconfiguration -n demo sample-nats-backup --type="merge" --patch='{"spec": {"paused": false}}'
+backupconfiguration.stash.appscode.com/sample-nats-backup patched
+```
+
+Verify that the `BackupConfiguration` has been resumed,
+```bash
+❯ kubectl get backupconfiguration -n demo sample-nats-backup
+NAME                 TASK                SCHEDULE      PAUSED   AGE
+sample-nats-backup   nats-backup-2.4.0   */2 * * * *   true     2d19h
+```
+
+Here,  `false` in the `PAUSED` column means the backup has been resumed successfully. The CronJob also should be resumed now.
+
+```bash
+❯ kubectl get cronjob -n demo
+NAME                               SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+stash-backup-sample-nats-backup   */5 * * * *   False     0        3m24s           4h54m
+```
+
+Here, `False` in the `SUSPEND` column means the CronJob is no longer suspended and will trigger in the next schedule.
+
+### Restore Into Different NATS Cluster of the Same Namespace
+
+If you want to restore the backed up data into a different NATS Cluster of the same namespace, you have to create another `AppBinding` pointing to the desired NATS Cluster. Then, you have to create the `RestoreSession` pointing to the new `AppBinding`.
+
+### Restore Into Different Namespace
+
+If you want to restore into a different namespace of the same cluster, you have to create the Repository, backend Secret, AppBinding, in the desired namespace. You can use [Stash kubectl plugin](https://stash.run/docs/latest/guides/latest/cli/cli/) to easily copy the resources into a new namespace. Then, you have to create the `RestoreSession` object in the desired namespace pointing to the Repository, AppBinding of that namespace.
+
+### Restore Into Different Cluster
+
+If you want to restore into a different cluster, you have to install Stash in the desired cluster. Then, you have to install Stash NATS addon in that cluster too. Then, you have to create the Repository, backend Secret, AppBinding, in the desired cluster. Finally, you have to create the `RestoreSession` object in the desired cluster pointing to the Repository, AppBinding of that cluster.
```

Comment:
Remove this line.
`Then, you have to install Stash NATS addon in that cluster too.`